External/Plugins/ASCompletion/Completion/ASComplete.cs

```diff
@@ -414,27 +416,42 @@ public static void HandleAddClosingBraces(ScintillaControl sci, char c, bool add
                         HandleRemoveBrace(sci, c, braces);
                 }
             }
+            // If the current style is a string literal in double/single quotes and the matching char is entered
+            else if (c == '"' && IsDoubleQuoteStyle(style) || c == '\'' && IsSingleQuoteStyle(style))
+            {
+                // If the next char is the corresponding closing char
+                if (addedChar && c == sci.CurrentChar)
```

Comment:
The styles are in fact defined in `PluginCore.ScintillaNet.Lexers`. AS/Haxe typically use the Cpp lexer.